src/foam/layout/SectionAxiom.js

```diff
@@ -52,17 +52,20 @@ foam.CLASS({
       });
 
       if ( this.permissionRequired ) {
-        var permSlot = data$.map(data => {
-          if ( ! data || ! data.__subContext__.auth ) return false;
-          return data.__subContext__.auth.check(null,
-            `${data.cls_.id.toLowerCase()}.section.${this.name}`);
-        });
+        var permSlot = foam.core.SimpleSlot.create({value: false});
+        var data = data$.get();
+        if ( data && data.__subContext__.auth ) {
```

Comment:
This block won't update when data changes now though. If `data` is null at first and gets set later, the permission check won't be performed.